platform/settings.cpp

```diff
@@ -17,375 +17,407 @@
 #include "std/iostream.hpp"
 #include "std/sstream.hpp"
 
-
 #define FIRST_LAUNCH_KEY "FirstLaunchOnDate"
 
-static char const DELIM_CHAR = '=';
+namespace
+{
+char const DELIM_CHAR = '=';
```

Comment:
You can't reformat it in this request as it already had been merged. But you promised to consider all my concerns regarding already merged request.